packages/core/src/components/Dropdown/components/ChildrenContent/ChildrenContent.jsx

```diff
@@ -36,7 +37,9 @@ export const ChildrenContent = ({ data, children, readOnly }) => {
           {data?.leftIcon && <Icon iconSize={18} icon={data.leftIcon} clickable={false} />}
         </>
       )}
-      {children}
+      <Text type="text2" color="inherit">
```

Comment:
Is it working and not throwing errors? we've only allowed this at the Vibe3 branch, didn't we?

Reply:
Weird that it didn't throw any errors, changed it